doc/sphinx/source/howto/derive-for-training.rst

```diff
@@ -24,27 +25,64 @@ MNIST data.
   - :ref:`update`
 
 
+.. _understanding_ml_ecosystem:
+
+Understanding the ML ecosystem
+===============================
+
+In a :abbr:`Machine Learning (ML)` ecosystem, it makes sense to take advantage 
+of automatable tasks as much as possible. As such, nGraph was designed to use 
+all available "automatic" graph construction endpoints handed down to it from a 
+framework. Our graph-construction API, therefore, needs to operate at a 
+fundamentally lower level than a typical framework's API. And, due to the fact
+that the nGraph :abbr:`Intermediate Representation (IR)` layer is what handles 
+the translation from the framework's endpoints to an optimized (optimizable) ML 
+environment, writing a model directly in nGraph would be somewhat akin to 
+programming in assembly language: not impossible, but not exactly the easiest
+thing for people to do. 
+
+Yet, when choosing among topologies for development, it's important to not 
+lose sight of the ultimate deployability and machine-runtime demands of a model 
+that you ultimately want trained. Likeise, when developing a new *genre* for ML/DL 
```

Comment:
Typo (`Likeise`)

Reply:
I have some comments below on the use of the word "genre", which is repeated elsewhere.

Reply:
Thanks, this section doesn't belong on this page, it is what I was working on to add to the index.